dipy/viz/actor.py

```diff
@@ -0,0 +1,648 @@
+# from __future__ import division, print_function, absolute_import
+
+import numpy as np
+
+from dipy.viz.colormap import line_colors
+from dipy.viz.utils import numpy_to_vtk_points, numpy_to_vtk_colors
+from dipy.viz.utils import set_input, map_coordinates_3d_4d
+from dipy.core.ndindex import ndindex
+
+# Conditional import machinery for vtk
+from dipy.utils.optpkg import optional_package
+
+# Allow import, but disable doctests if we don't have vtk
+vtk, have_vtk, setup_module = optional_package('vtk')
+colors, have_vtk_colors, _ = optional_package('vtk.util.colors')
+numpy_support, have_ns, _ = optional_package('vtk.util.numpy_support')
+
+if have_vtk:
+
+    version = vtk.vtkVersion.GetVTKSourceVersion().split(' ')[-1]
+    major_version = vtk.vtkVersion.GetVTKMajorVersion()
+
+
+def slicer(data, affine=None, value_range=None, opacity=1.,
+          lookup_colormap=None):
+    """ Cuts 3D scalar or rgb volumes into images
+
+    Parameters
+    ----------
+    data : array, shape (X, Y, Z) or (X, Y, Z, 3)
+        A grayscale or rgb 4D volume as a numpy array.
+    affine : array, shape (3, 3)
+        Grid to space (usually RAS 1mm) transformation matrix
+    value_range : None or tuple (2,)
+        If None then the values will be interpolated to (0, 255) from
```

Comment:
What is this parameter? Is it the range into which the values will be interpolated? Condensed?